src/services/api.tsx

```diff
@@ -0,0 +1,57 @@
+import { URL_SERVER } from "../constants";
+import { Patient } from "types";
+
+const mkFhir = require("fhir.js");
+const client = mkFhir({
+  baseUrl: URL_SERVER
+});
+
+function getAge(birthDate: Date) {
+  const today = new Date();
+  const monthDifference = today.getMonth() - birthDate.getMonth();
+
+  const age = today.getFullYear() - birthDate.getFullYear();
+  if (
+    monthDifference < 0 ||
+    (monthDifference === 0 && today.getDate() < birthDate.getDate())
+  )
+    return age - 1;
+  return age;
+}
+
+const callApi = (resourceType: string, queryParameters: object) => {
+  const patient = client.search({
+    type: resourceType,
+    query: queryParameters
+  });
+  return patient;
+};
+
+export const getPatients = () => {
+  const fetchPatients = async () => {
```

Comment:
Why do you redefine an async function within the `getPatients` function ? I believe you don't need to